comment/templates/comment/email/footer.html

```diff
@@ -0,0 +1,12 @@
+<table role="presentation" border="0" cellpadding="0" cellspacing="0">
+    <tr>
+        <td class="content-block">
+            <span class="apple-link">Company Inc, 3 Abbey Road, San Francisco CA 94102</span>
```

Comment:
Do we need to include the company address for here?

Reply:
😄 I leave it to the developer to change it, even if we use our custom one, the developer will definitely change it to their need so why bother to change it?

Reply:
> the developer will definitely change it to their need so why bother to change it?

I do get your point.

They should but that doesn't mean they would :smile: They are people like me who aren't as attentive to these small details as others. In my opinion, it might be better to not include these here as some may forget to change it and see it later.

Reply:
You are right, I will disable the footer and it can be added optionally.
I also was thinking of implementing unsubscribe view but I saw the current PR is already giant.

Reply:
too giant to say the least. please don't :bow: . I already see an unnecessary change log also in there.